vcd/resource_vcd_vapp_vm.go

```diff
@@ -381,6 +381,13 @@ func vmSchemaFunc(vmType typeOfVm) map[string]*schema.Schema {
 			Optional: true,
 			Set:      resourceVcdVmIndependentDiskHash,
 		},
+		"consolidate_disks_on_create": {
+			Type:        schema.TypeBool,
+			Optional:    true,
+			Default:     false,
+			ForceNew:    true,
```

Comment:
Why the `ForceNew`? My understanding is that this property is only used during creation. It should not make a difference if it is modified later on.

Reply:
It wouldn't, but if there is no `ForceNew`, it would make false impression of update doing something.